pom.xml

```diff
@@ -1734,375 +1774,386 @@
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-clean-plugin</artifactId>
-        <version>2.3</version>
-                <configuration>
-                  <filesets>
-                    <fileset>
-                      <directory>data/.myrobotlab</directory>
-                      <followSymlinks>false</followSymlinks>
-            </fileset>
-          <fileset>
-            <directory>libraries</directory>
-            <includes>
-              <include>**</include>
-            </includes>
-            <followSymlinks>false</followSymlinks>
-          </fileset>
-          <fileset>
-            <directory>data</directory>
-            <includes>
-            <include>**</include>
-            </includes>
-          </fileset>
-          <fileset>
-            <directory>resource</directory>
-            <includes>
-            <include>**</include>
-            </includes>
-          </fileset>
-          <fileset>
-            <directory>src/main/resources/resource/framework</directory>
-            <includes>
-              <include>**/serviceData.json</include>
-            </includes>
-            <followSymlinks>false</followSymlinks>
-          </fileset>
-                  </filesets>
-                </configuration>
-      </plugin>
-      <!-- attached to Maven test phase -->
-      <!--
-      <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.8.8</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>prepare-agent</goal>
-            </goals>
-          </execution>          
-          <execution>
-            <id>report</id>
-            <phase>test</phase>
-            <goals>
-              <goal>report</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      -->
-    </plugins>
-  </build>
-    <reporting>    
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-report-plugin</artifactId>
-        <version>2.22.2</version>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-javadoc-plugin</artifactId>
-        <version>3.0.1</version>
-      </plugin>
-    </plugins>
-    </reporting>
-  <organization>
-    <name>myrobotlab</name>
-    <url>http://myrobotlab.org</url>
-  </organization>
-  <issueManagement>
-      <system>github</system>
-      <url>https://github.com/MyRobotLab/myrobotlab/issues</url>
-  </issueManagement>
-</project>
+<dependency>
+    <groupId>dev.onvoid.webrtc</groupId>
+    <artifactId>webrtc-java</artifactId>
+    <version>0.7.0</version>
+</dependency>
+
+
+  	<dependency>
+	    <groupId>org.mockito</groupId>
+	    <artifactId>mockito-core</artifactId>
+	    <version>3.12.4</version>
+	    <scope>test</scope>
+	</dependency>
+  	<dependency>
```

Comment:
the pom for this is missing in the mrl repo so it doesn't cache properly when doing offline builds.

Reply:
you can see the issue when running a mvn build from the command line with the following error message.

[WARNING] The POM for au.edu.federation.caliko.visualisation:caliko-visualisation:jar:1.3.8 is missing, no dependency information available
[WARNING] The POM for au.edu.federation.caliko.demo:caliko-demo:jar:1.3.8 is missing, no dependency information available